### ISSUE TITLE
General Community Outreach

### ISSUE THREAD

**Issue by ligthyear** [kentcdodds/all-contributors#34]
I see there are a number of emoji about specific contributions done for the projects, which are about involving more people (giving a talk, writing a blog post, etc) and I like that a lot – as this is about shining light on exactly those often overlooked contributions. Yet, by making those so specific, it becomes harder to show more general communication efforts of people – like a person super responsive on the chat or organising the issues or organising user groups or other gathering events for the team and community around it – who are very often not in the spotlight despite the hours of hours they just chuck away for teh project.

I propose to have a "General Community Outreach" work emoji, that can be used as a placeholder for all the great community work done by people but not captured by the emojis so far.

maybe 👥 or the awesome 🕴️(MAN IN BUSINESS SUIT LEVITATING) or [any of the thousand family icons](http://unicode.org/emoji/charts/full-emoji-list.html#1f46a)?

**Comment by skade** [comment]
Hm, I'm not sure if I would fold that into one emoji, running meetups and doing moderation are some quite different things.

**Comment by kentcdodds** [comment]
I agree with @skade here... I think that I want to avoid putting really broadly defined items into the all-contributors spec. I think it's reasonable to expect that projects will customize the spec to suit their needs. But by putting in broad contribution items, it makes things much more subjective and difficult to know what to apply for a certain type of contribution...

Forgive me if I'm not making sense. It's late for me right now :-)

**Comment by ligthyear** [comment]
Fair enough. Then I'd propose to add:

 - Organising Community Events: 🍻 (is the only one a little bit communal, though I don't like the alcohol reference... 🍽️ is just way less community, or maybe one of the party icons, like 🎉)
 - Overseeing Issues/Bug Reports: 🗃️ (card file box)
 - Project Coordination 🌐
 - (personal) Coaching/Mentoring: 🏫
 - Money Management/Fund Raising/Sponsoring: 💸

**Comment by skade** [comment]
* :microphone: seems to be a goods suggestion for events.

**Comment by skade** [comment]
Lots of suggestions here: https://twitter.com/Argorak/status/707858755489304576

**Comment by Jameskmonger** [comment]
I like :calendar: or :date: @skade

**Comment by skade** [comment]
Both look very neat.

I'd also like to add "Moderation", maybe using 🗣 or 💬?

**Comment by Jameskmonger** [comment]
That first emoji doesn't work on my PC @skade. Could we stick to using GitHub emojis where possible? They're always supported, unlike the unicode ones which aren't

**Comment by skade** [comment]
Hm, nice catch, the first is not in the github emoji ranges. *grumbles something about standards*

**Comment by kentcdodds** [comment]
Thanks for the discussion! What I like:

- Organising Community Events:  🎉
- Overseeing Issues/Bug Reports: Let's update the "Reviewed Pull Requests" to account for this as well which is 👀
- Project Coordination 🌐
- (personal) Coaching/Mentoring: 🏫
- Money Management/Fund Raising/Sponsoring: 💸

:+1:

**Comment by ligthyear** [comment]
Ok. this becomes a little fuzzy now. What is the process to decide on emoji if there are multiple contestor? Can we use the new reaction-emoji-feature of github to do a voting? How many votes are needed/who can vote?

**Comment by kentcdodds** [comment]
There's no set rules, but I'll make some up right now...

You need at least 2 :+1: votes and the :+1: votes need to outweigh the :-1: votes. People can also just say they don't care either way and that's a neutral vote (doesn't count either way). Sound good?

**Comment by tommck** [comment]
I'm too old to like emoji for things like this.. I will invariably look for text that will describe what it's supposed to mean

**Comment by mk-pmb** [comment]
> text that will describe what it's supposed to mean

That would help me as well. Does the link syntax support a title attribute? Then we could use that to explain. I can't currently find any explanation for the :tipping_hand_woman: in the readme.

I'd prefer not to be listed as contributor just for this comment. Also there should be a shorthand (emoji?) to state the previous. ;-)

**Comment by kentcdodds** [comment]
> Does the link syntax support a title attribute? Then we could use that to explain.

Yes, that would be useful. I'm sure that could be done...

>  I can't currently find any explanation for the 💁 in the readme.

That's because it was changed (because it's confusing) to 💬 in [v2.0.0](https://github.com/kentcdodds/all-contributors/releases/tag/v2.0.0)

> I'd prefer not to be listed as contributor just for this comment. Also there should be a shorthand (emoji?) to state the previous. ;-)

Most of the time, I ask others to make a PR to list themselves, and if I do make the PR myself, I ask before merging. :)

**Comment by Berkmann18** [comment]
Since this issue was opened the [emoji key](https://allcontributors.org/docs/en/emoji-key) had some additions (e.g: PM, maintainer etc) and I was wondering what do you guys want to add?
I like the idea of having the following categories: moderation (the 💬 is taken by `question`), coaching/mentoring (🏫).
